cwms-data-api/src/main/java/cwms/cda/data/dao/RatingMetadataDao.java

```diff
@@ -64,11 +78,13 @@ public class RatingMetadataDao extends JooqDao<RatingSpec> {
             + "<ratings xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" "
             + "xsi:noNamespaceSchemaLocation=\"https://www.hec.usace.army"
             + ".mil/xmlSchema/cwms/Ratings.xsd\"/>";
-    private final Executor executor = Executors.newCachedThreadPool(r -> {
-        Thread thread = new Thread(r, getClass().getSimpleName());
-        thread.setDaemon(true);
-        return thread;
-    });
+    private final Executor executor = new ThreadPoolExecutor(6, Integer.MAX_VALUE,
```

Comment:
You know, technically I think since this is a Java EE webapp we should be getting our thread pools from the server itself.

I don't want to deal with it now, but it is something to think about.